node/bin/src/config.rs

```diff
@@ -432,6 +434,39 @@ pub struct LogConfig {
     pub use_color: bool,
 }
 
+/// Configuration for batch verification client and server
+#[derive(Clone, Debug, DescribeConfig, DeserializeConfig)]
+#[config(derive(Default))]
+pub struct BatchVerificationConfig {
+    /// [server] If we are collecting batch verification signatures
+    /// [en] If we are signing batches
+    #[config(default_t = false)]
+    pub enabled: bool,
+    /// [server] Batch verification server address to listen on.
+    /// [en] Batch verification server address to connect to.
+    #[config(default_t = "0.0.0.0:3072".into())]
+    pub address: String,
+    /// [server] Threshold (number of needed signatures)
+    #[config(default_t = 1)]
+    pub threshold: usize,
+    /// [server] Accepted signer pubkeys
+    #[config(default_t = vec!["0x36615Cf349d7F6344891B1e7CA7C72883F5dc049".into()])]
+    pub accepted_signers: Vec<String>,
```

Comment:
Do we want to make it a part of the local configuration? I assume that later it will be taken from some other source, e.g. L1, but if we introduce it as a part of overall node configuration, it might be hard to remove later.

Probably question for @RomanBrodetski -- do we want any kind of notion for experimental/temporary configs that do not fall under the breaking changes rules?